frontend/app/component/form/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -67,6 +71,7 @@ describe("Test PollingStationChoiceForm", () => {
   });
 
   test("Polling station list", async () => {
+    overrideOnce("get", "/api/polling_stations/1", 200, pollingStationMock);
```

Comment:
```suggestion
    overrideOnce("get", "/api/elections/1/polling_stations", 200, pollingStationMock);
```

Reply:
Thanks, I've changed the overrides in other tests as well, since they're actually all calling the listing endpoint.